lib/rubocop/cop/style/request_referer.rb

```diff
@@ -0,0 +1,38 @@
+# encoding: utf-8
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # This cop checks for uses of request.referrer.
+      class RequestReferer < Cop
+        include ConfigurableEnforcedStyle
+
+        def on_send(node)
+          if offense?(node)
+            add_offense(node.source_range, node.source_range, message)
+          end
+        end
+
+        def autocorrect(node)
+          ->(corrector) { corrector.replace(node, "request.#{style}") }
+        end
+
+        private
+
+        def offense?(node)
+          receiver = node.receiver
```

Comment:
Thanks, I'll take a look into this. It's my first time with Rubocop code, I'll try to figure out a better way